pallets/estate/src/lib.rs

```diff
@@ -375,4 +362,37 @@ impl<T: Config> Module<T> {
 		})?;
 		Ok(estate_id)
 	}
+
+	fn mint_land_unit(
```

Comment:
Should it called mint_land_units instead, as it accept multiple coordinates?

Reply:
updated